.github/workflows/release.yml

```diff
@@ -15,6 +15,15 @@ on:
         type: boolean
         default: true
         description: "Flag that controls whether the box upload should be done."
+      box_root_folder_name:
+        required: true
```

Comment:
If box_upload is false than this input is not required

Reply:
fixed